packages/components/src/chip/src/Chip.module.css

```diff
@@ -0,0 +1,238 @@
+}
+
+.hop-Chip--neutral {
+    --border-color: var(--hop-Chip-neutral-border-color);
+    --background-color: var(--hop-Chip-neutral-background-color);
+    --text-color: var(--hop-Chip-neutral-text-color);
+}
+
+.hop-Chip--progress {
+    --border-color: var(--hop-Chip-progress-border-color);
+    --background-color: var(--hop-Chip-progress-background-color);
+    --text-color: var(--hop-Chip-progress-text-color);
+}
+
+.hop-Chip--positive {
+    --border-color: var(--hop-Chip-positive-border-color);
+    --background-color: var(--hop-Chip-positive-background-color);
+    --text-color: var(--hop-Chip-positive-text-color);
+}
+
+.hop-Chip--caution {
+    --border-color: var(--hop-Chip-caution-border-color);
+    --background-color: var(--hop-Chip-caution-background-color);
+    --text-color: var(--hop-Chip-caution-text-color);
+}
+
+.hop-Chip--negative {
+    --border-color: var(--hop-Chip-negative-border-color);
+    --background-color: var(--hop-Chip-negative-background-color);
+    --text-color: var(--hop-Chip-negative-text-color);
+}
+
+.hop-Chip--option1 {
+    --border-color: var(--hop-Chip-option1-border-color);
+    --background-color: var(--hop-Chip-option1-background-color);
+    --text-color: var(--hop-Chip-option1-text-color);
+}
+
+.hop-Chip--option2 {
+    --border-color: var(--hop-Chip-option2-border-color);
+    --background-color: var(--hop-Chip-option2-background-color);
+    --text-color: var(--hop-Chip-option2-text-color);
+}
+
+.hop-Chip--option3 {
+    --border-color: var(--hop-Chip-option3-border-color);
+    --background-color: var(--hop-Chip-option3-background-color);
+    --text-color: var(--hop-Chip-option3-text-color);
+}
+
+.hop-Chip--option4 {
+    --border-color: var(--hop-Chip-option4-border-color);
+    --background-color: var(--hop-Chip-option4-background-color);
+    --text-color: var(--hop-Chip-option4-text-color);
+}
+
+.hop-Chip--option5 {
+    --border-color: var(--hop-Chip-option5-border-color);
+    --background-color: var(--hop-Chip-option5-background-color);
+    --text-color: var(--hop-Chip-option5-text-color);
+}
+
+.hop-Chip--option6 {
+    --border-color: var(--hop-Chip-option6-border-color);
+    --background-color: var(--hop-Chip-option6-background-color);
+    --text-color: var(--hop-Chip-option6-text-color);
+}
+
+.hop-Chip--inactive {
+    --border-color: var(--hop-Chip-inactive-border-color);
+    --background-color: var(--hop-Chip-inactive-background-color);
+    --text-color: var(--hop-Chip-inactive-text-color);
+}
+
+.hop-Chip--disabled {
+    --border-color: var(--hop-Chip-disabled-border-color);
+    --background-color: var(--hop-Chip-disabled-background-color);
+    --text-color: var(--hop-Chip-disabled-text-color);
+}
+
+.hop-Chip__icon,
+.hop-Chip__icon-list {
+    flex: 0 0 auto;
+    flex-wrap: nowrap;
+    order: 1;
+}
+
+.hop-Chip__text {
+    overflow: hidden;
+    flex: 1 1 auto;
+    order: 2;
+
+    min-inline-size: 0;
+
+    text-align:center;
```

Comment:
`text-align: center;`

Reply:
ya, hmm I feel like this should be in the linting rules..